styles/_base.scss

```diff
@@ -103,6 +103,12 @@ ul, ol {
   background-repeat: no-repeat;
 }
 
+.trumbowyg .trumbowyg-editor {
```

Comment:
Would this be more appropriate inside of components/misc? It seems like a really random thing to be inside of base.